dev-packages/browser-integration-tests/suites/public-api/instrumentation/onError/thrown-strings/subject.js

```diff
@@ -0,0 +1,5 @@
+function throwStringError() {
+  throw 'stringError';
+}
+
+throwStringError();
```

Comment:
The problem with this code is that it throws an error immediately when the subject script is evaluated by the browser. This results in the infamous `Script Error` which the SDK filters by default. If you enable debug logging you should see that the error gets dropped by the SDK. 

I usually add a click listener to a button and click it in the test to work around this.